spec/requests/api/v3/priority_resource_spec.rb

```diff
@@ -55,5 +57,61 @@
 
       it_behaves_like 'API V3 collection response', 2, 2, 'Priority'
     end
+
+    context 'not logged in user' do
+      before do
+        get get_path
+      end
+
+      it_behaves_like 'error response',
+                      403,
+                      'MissingPermission',
+                      I18n.t('api_v3.errors.code_403')
+    end
+  end
+
+  describe 'priorities/:id' do
+    subject(:response) { last_response }
+
+    let(:get_path) { "/api/v3/priorities/#{priorities.first.id}" }
+
+    context 'logged in user' do
+      before do
+        allow(User).to receive(:current).and_return current_user
+        member = FactoryGirl.build(:member, user: current_user, project: project)
+        member.role_ids = [role.id]
+        member.save!
```

Comment:
@Azure7111 @NobodysNightmare 

Not an issue, just a pointer -> there is a more condensed way of writing this: https://github.com/opf/openproject/blob/dev/spec/models/query/results_spec.rb#L71